src/App.tsx

```diff
@@ -1,61 +1,133 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { PreparedTodo, Todo } from './interfaces/Todo';
+import { User } from './interfaces/User';
+import { TodoList } from './components/TodoList';
+
+const findUser = (users: User[], todoUserId: number): User | null => {
+  return users.find((user: User) => user.id === todoUserId) || null;
+};
+
+const userJoinToTodos = (todos: Todo[]): PreparedTodo[] => {
+  return todos
+    .map((todo: Todo) => ({
+      ...todo,
+      user: findUser(usersFromServer, todo.userId),
+    }));
+};
+
+const preparedTodos: PreparedTodo[] = userJoinToTodos(todosFromServer);
```

Comment:
I think the `userJoinToTodos` function is unnecessary here
```suggestion
const preparedTodos: PreparedTodo[] = todos
    .map((todo: Todo) => ({
      ...todo,
      user: findUser(usersFromServer, todo.userId),
    }));
```